Source/WebCore/inspector/InspectorOverlayLabel.cpp

```diff
@@ -239,7 +239,7 @@ Path InspectorOverlayLabel::draw(GraphicsContext& context, float maximumLineWidt
                 text = makeString(text, ellipsis);
                 while (currentLineWidth + textWidth + (labelPadding * 2) > maximumLineWidth && text.length() > 1) {
                     // Remove the second from last character (the character before the ellipsis) and remeasure.
-                    text.remove(text.length() - 2);
+                    text = text.left(text.length() - 2);
```

Comment:
Is it correct? I think the original code was only removing one second-from-last character (the last character is remaining).

Reply:
Oh, you're right, I am removing the 2 last characters and the previous code was removing only the second to last. Will fix.